Branches/Stable/Behaviors/obj_Miner.iss

```diff
@@ -1669,7 +1669,7 @@ objectdef obj_Miner
 					UI:UpdateConsole["Debug: Entity:WarpTo to ${LocationID} from Line _LINE_ ", LOG_DEBUG]
 					Entity[${LocationID}]:WarpTo[0]
 				}
-				if ${Universe[${LocationID}](exists)}
+				if ${Universe[${LocationID}](exists)} && ${Universe[${LocationID}].Name} != NULL

```

Comment:
I used this
UI:UpdateConsole["Debug: Universe:SetDestination to ${LocationID} - name: ${Universe[${LocationID}].Name} from Line _LINE_ ", LOG_DEBUG]

and it printed NULL